.github/workflows/benchmark-pg_search.yml

```diff
@@ -41,25 +31,45 @@ jobs:
   benchmark-pg_search:
     name: Benchmark pg_search
     runs-on: ubuntu-latest-m # Size: 4-cores · 16 GB RAM · 150 GB SSD
+    env:
+      pg_version: 16
 
     steps:
       - name: Checkout Git Repository
         uses: actions/checkout@v4
 
-      - name: Configure System to Benchmark
-        id: system
-        run: echo "system_to_benchmark=${{ github.event.inputs.name || 'pg_search' }}" >> $GITHUB_OUTPUT
+      - name: Install & Configure Supported PostgreSQL Version
+        run: |
+          wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+          sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+          sudo apt-get update && sudo apt-get install -y postgresql-${{ env.pg_version }} postgresql-server-dev-${{ env.pg_version }}
+          sudo chown -R $(whoami) /usr/share/postgresql/${{ env.pg_version }}/extension/ /usr/lib/postgresql/${{ env.pg_version }}/lib/ /var/lib/postgresql/${{ env.pg_version }}/ /usr/lib/postgresql/${{ env.pg_version }}/bin/
+          echo "/usr/lib/postgresql/${{ env.pg_version }}/bin" >> $GITHUB_PATH
+
+      - name: Install pgrx & pg_search
+        working-directory: pg_search/
+        run: |
+          cargo install -j $(nproc) --locked cargo-pgrx --version 0.11.3
+          cargo pgrx init --pg${{ env.pg_version }}=/usr/lib/postgresql/${{ env.pg_version }}/bin/pg_config
+          cargo pgrx install
 
-      # The '-t local' tag is used for benchmark-pg_search.sh and ignored by other scripts
-      - name: Run Benchmarking Script
-        working-directory: pg_search/benchmarks/
-        run: ./benchmark-${{ steps.system.outputs.system_to_benchmark }}.sh -t local
+      - name: Add pg_search to shared_preload_libraries
+        working-directory: /home/runner/.pgrx/data-${{ env.pg_version }}/
+        run: sed -i "s/^#shared_preload_libraries = .*/shared_preload_libraries = 'pg_search'/" postgresql.conf
 
-      - name: Print Results
-        working-directory: pg_search/benchmarks/out/
-        run: cat benchmark_${{ steps.system.outputs.system_to_benchmark }}.csv
+      - name: Install the ParadeDB Benchmarking Tool
+        working-directory: cargo-paradedb/
+        run: cargo run install
+
+      - name: Run pg_search Benchmarks
+        working-directory: pg_search/
+        run: |
+          cargo pgrx start
+          cargo paradedb bench eslogs generate --url postgresql://localhost:288${{ env.pg_version }}/postgres
```

Comment:
@neilyio here I just ran the defaults, should we run on a larger dataset? Feel free to edit the commands here to make the benchmarks more realistic

Reply:
Yes I added `--events 5000000` to match the benchmark script that we replaced. Thank you for doing this!